storage/src/vespa/storage/persistence/filestorage/filestorhandlerimpl.cpp

```diff
@@ -14,9 +14,11 @@
 #include <vespa/storageapi/message/stat.h>
 #include <vespa/vespalib/stllike/hash_map.hpp>
 #include <vespa/vespalib/util/exceptions.h>
+#ifndef XXH_NO_INLINE_ALL
```

Comment:
Since we are controlling this in the build_settings.cmake I think the whole XXH_INLINE stuff here can just go away.